src/js/client.ts

```diff
@@ -204,4 +205,24 @@ export class ReactNativeClient extends BaseClient<ReactNativeClientOptions> {
       envelope[items].push(clientReportItem);
     }
   }
+
+  /**
+   * If enabled attaches a screenshot to the event hint.
+   */
+  private _attachScreenshotToEventHint(hint: EventHint): PromiseLike<EventHint> {
```

Comment:
M: Does it make sense to be an integration or event processor instead?
If not, that's ok, but something we have to do is to at least add an integration in the `sdk.integrations` list so we can track adoption of this feature.
Only if `attachScreenshot` is enabled of course.

Reply:
As the order of the integrations is not guaranteed, and integrations can be async the screen could be captured too late. Or would that still be good enough? I know other platforms also use the event processor for the screenshots.

I'll add the Integration so we can track the adoption.